topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/tutorial.md

```diff
@@ -0,0 +1,687 @@
+---
+layout: tutorial_hands_on
+
+title: 'Trajectory Analysis using Monocle3 '
+subtopic: single-cell-CS
+priority: 5
+zenodo_link: 'https://zenodo.org/record/7078524'
+
+questions:
+- How to prepare input files for Monocle starting from AnnData object?
+- How can I infer lineage relationships between clusters, without a time series?
+- What trajectory analysis can tell us?
+objectives:
+- Understand which opertions to perform on AnnData object to obtain files needed for Monocle
+- Follow the Monocle3 workflow and choose the right parameter values
+- Compare output from Scanpy and Monocle
+- Interpet trajectory analysis results
+
+time_estimation: 1H
+
+key_points:
+- You should understand your data to be able to extract relevant information for further analysis.
+- Trajectory analysis is highly dependent on the parameter values you choose, as such ‘inferred relationships’ are a bigger mathematical leap. Therefore, you should always check if the output makes biological sense before processing to the next step.
+- Comparing the output of two different methods applied on the same dataset might be useful to confirm the results, ensure that the findings are reliable and even sometimes find a new piece of information.
+
+requirements:
+-
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-case_alevin
+        - scrna-case_alevin-combine-datasets
+        - scrna-case_basic-pipeline
+        - scrna-case_JUPYTER-trajectories
+tags:
+- single-cell
+- trajectory-analysis
+- paper-replication
+
+contributors:
+- wee-snufkin
+- nomadscientist
+
+---
+
+# Introduction
+
+This tutorial is a follow-up to the 'Single-cell RNA-seq: Case Study' (find it [here](https://training.galaxyproject.org/training-material/topics/transcriptomics/)), we will use the same sample from the previous tutorials. If you haven’t done them yet, it’s highly recommended that you go through them to get an idea how to [prepare a single cell matrix](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-case_alevin/tutorial.html), [combine datasets](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.html) or [filter, plot and process scRNA-seq data](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-case_basic-pipeline/tutorial.html) to get the data in the form we’ll be working on today.
```

Comment:
```suggestion
This tutorial is a follow-up to the ['Single-cell RNA-seq: Case Study']({% link topics/transcriptomics/index.md %}), we will use the same sample from the previous tutorials. If you haven’t done them yet, it’s highly recommended that you go through them to get an idea how to [prepare a single cell matrix]({% link topics/transcriptomics/tutorials/scrna-case_alevin/tutorial.md %}), [combine datasets]({% link topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md %}) or [filter, plot and process scRNA-seq data]({% link topics/transcriptomics/tutorials/scrna-case_basic-pipeline/tutorial.md %}) to get the data in the form we’ll be working on today.
```

The only link checking we do regularly is internal link checking (as external is extremely time consuming and error prone). 

By converting the tutorial links to internal links we ensure that they get checked automatically, and that if someone hosts this on their own domain, it stays on the same site.

Reply:
You can probably bulk replace these with something like 

```
sed -i "s|(https://training.galaxyproject.org/training-material/\([^)]*\).html)|{% link \1.md %} path/to/tutorial.md
```